apps/web/frontend/components/projectInvitation.tsx

```diff
@@ -0,0 +1,85 @@
+import { useRef } from 'react'
+import { BiCopyAlt } from 'react-icons/bi'
+import { useMutation } from 'urql'
+
+import { toastSuccess } from '@progwise/timebook-ui'
+
+import { graphql } from '../generated/gql'
+
+export const ProjectInvitationFragment = graphql(`
+  fragment ProjectInvitationFragment on Project {
+    id
+  }
+`)
+
+const projectMembershipInvitationMutation = graphql(`
+  mutation projectMembershipInvitation($projectId: ID!) {
+    projectMembershipInvitationCreate(projectId: $projectId) {
+      id
+      invitationKey
+      expireDate
```

Comment:
Can we display the expire Date in the dialog, maybe with https://date-fns.org/v3.6.0/docs/formatDistanceToNow